### ISSUE TITLE
Maximo User Process Discovery - Utilities & Structures UER

### ISSUE THREAD

**Issue by ChristinaTremel** [cityofaustin/atd-data-tech#16125]
## Objective 
The Data & Technology Services Division is initiating a series of meetings to learn more about Maximo user experiences and processes in TPW. In this meeting we'll be asking questions about how individual teams use Maximo and how it fits into their work processes.

## Goals:  
- Maximo system performance and functionality 
- User experience and interface experiences 
- Current processes within and related to Maximo 
- User feedback and improvement  
- Future roadmap and upgrades 

### User experience and interface experiences 
- Explain your process on how you use Maximo? 

### PROCESS - Planning
- Who plans the work? How do you get work to do? 
- How is work scheduled and issued out 
 
### PROCESS - Implementation
- How is work scheduled and issued out?
- Who is executing the implementation? 
 

### PROCESS - Field Work
- Who is executing the work - implementation? 
- Who is reporting on the work and what are they tracking? 

#### Maximo system performance and functionality 
- How long is it taking to do a Maximo work order in the field? 

### PROCESS - Review & Reporting
- What is your final review, QC process? 
- What reporting do you care about? How are you pulling that information out? 
- What are you reporting? 

### User feedback and improvement  
- What challenges are you having with Maximo? 

### Future roadmap and upgrades 
- What would you like your work order management system to do that you’re not getting right now? What’s missing? 

### Points of Contacts 

------
- [x] Schedule meeting 
- [x] Meet and take notes
- [ ] Process notes on Miro [Maximo Assessment Plan](https://miro.com/app/board/uXjVN7sxj9I=/)

**Comment by ChristinaTremel** [comment]
# Christina's Notes
### User experience and interface experiences
#### Explain your process on how you use Maximo?
- Uses Maximo from start to finish
- Receives work from Austin Water as a Service Request
   - Service Requests come from Austin Water to Maximo
      - When request comes through, things are coded with Austin Water assets and Michael has to update these with SBO assets
      - Come in as 1NEW
      - Service Address and the Maximo number is on the parent
   - Michael sets up the work order, description, and gets the ticket ready for the Supervisor
      - Create children from the parent
      - Sometimes combine request in the same child and sometimes split it up into several child records
      - Creates child records based on how the work needs to happen
         - If there assets that don’t mesh with each other like concrete and asphalt, concrete has to happen first and cure, then asphalt can come in. That would require 2 work orders
         - If assets mesh together (like just concrete assets), that can be one child wor order and can be done all in one time
   - Submits ticket
   - Gets permit
   - 2APR for inspector: Inspector has to go check it out - inspector is within UER
        - Inspectors take a photo and once the photo is added, they can issue the work
   - 2HOLD: If they identify the asset incorrectly, they have to ask for a change order from Austin Water and this goes on hold
        - Adds a code to let Michael know what he needs
   - 2OKTS: inspection is complete and it’s ready to be scheduled
   - Asset change order that Austin Water approves if something was filled out incorrectly and once it’s approved does the work
   - Contractor work
   - Work ticket gets sent through Utility Cut group and Michael doesn’t see the work until the job is complete
- Almost all the work comes from Austin Water
   - They do occasionally get work from other teams (sealcoat, district maintenance assistance), but this team works for and is paid from Austin Water.
- Permitting Process:
   - If there’s not a permit, Michael has to get a permit
   - Has 30 days to get the work done on the permit that Austin Water pulls
      - The system with Austin Water is different
      - Michael gets tickets that have one day left on the permit and that’s not enough time to schedule the work
      - At that time, he has to pull a new permit
      - This has gotten better over time, but it could still be improved and some get lost in the shuffle.
   - If Michael has to pull a new permit, which takes him about 30 minutes on the portal. About a 15 page process
      - Gets the permit right away (typically within a couple days)
      - Critical arterials are more complicated
         - Takes more time
         - Requires engineer traffic control plan and that’s expensive
         - These permits take 3 months to pull
      - Have to provide a traffic control plan

### PROCESS - Planning
#### Who plans the work? How do you get work to do?
- Joe gets the locations for the next months (150-160), puts them in a map app and that’s how they schedule the work
- Map app on their iPad that shows scheduled stops and optimize routing
   - Circuit app used for that
#### How is work scheduled and issued out?
- Asphalt group does 8-10 work orders per day
   - 1 crew (10-12 people)
- Concrete group does anywhere from 7-8 per day
   - 1 big crew or 6 2-man crews
   - 6 trucks
- At the end of the day, the crews notify Joe what’s been done/not done and they have a weekly schedule that John checks and Joe manages
- Weekly schedule is tracked in a spreadsheet and Joe highlights things in green that have been completed on the schedule

### PROCESS - Implementation
#### How is work scheduled and issued out?
#### Who is executing the implementation?

### PROCESS - Field Work
#### Who is executing the work - implementation?
#### Who is reporting on the work and what are they tracking?
- When the crew comes back from the field they fill out the Maximo work order with the actuals
- John sees that the work order is green and he goes into Maximo and QAs the work order and closes it out in Maximo
- Mix use for Maximo work orders. Collect data using cell phone in the field: take photo, text lead when they are done with the job, fill out a travel log that’s in the truck, and text messages are used in Maximo for actuals
- When they come back, the crew lead enters the actuals in Maximo in the office
   - 2 crew leads in concrete
      - Lead will combine everything that’s been texted to them and communicated to them and fill out the work order back in the office using their laptop
      - They also fill out work orders sometimes in the field, but typically it’s when they get back in the office
    - Need to find out if this process is working for Maximo
    - Joe has heard that the iPads the technicians use are slow depending on where they are or the time of the day
- They use EzMax mobile

### Maximo system performance and functionality
#### How long is it taking to do a Maximo work order in the field?

### PROCESS - Review & Reporting
#### What is your final review, QC process?
#### What reporting do you care about? How are you pulling that information out?
#### What are you reporting?
- Monthly report is sent to Austin Water that reports what has been completed by the UER team. This is for reconciliation to charge for labor.
- Reporting hours, materials and that stuff to Finance.
   - All the staff in UER are billed directly through the task order number.
   - Supervisors sends tickets and receipts to finance for reconciliation for materials.
   - There’s a form filled out for the amount of materials that’s sent to finance

### User feedback and improvement
#### What challenges are you having with Maximo?
- Integration between Austin Water <> Maximo could use some improvements:
   - Sometimes service addresses don’t come in perfectly and the service address comes in as a long description
   - Sometimes the service address isn’t correct and it takes the team time to figure it out
   - Send the bill to purchasing weekly and then finance bills AWU monthly (uses UER Revenue Summary)
   - There’s a report that the supervisors pull to get counts of material

### Future roadmap and upgrades
####What would you like your work order management system to do that you’re not getting right now? What’s missing?

### Points of Contacts
Asphalt Leads: 
Concrete Leads:

**Comment by adry-martinez** [comment]
@dianamartin @ChristinaTremel @rebekkabry this discovery is completed, right?

**Comment by dianamartin** [comment]
[like]  Martin, Diana reacted to your message:
________________________________

**Comment by ChristinaTremel** [comment]
This is definitely completed, but @dianamartin did you have notes to add?

**Comment by dianamartin** [comment]
# 5/4/2024
## Diana's Notes

## DTS

- Christina, Rebekka, Rene, Diana

## UER
- John P. - UER Superintendent
- Joe H.  - Utility Cuts Supervisor
- Michael D.  - Project Coordinator

### PROCESS - Planning
Austin Water Service Request - ticket in Maximo

Issues with AW —> Maximo

- Not on actual address
- Service Address is missing
    - Look at long description
    - Find it in Maximo
    - Assign it as Service Address

- Physical address, - they get service addresses
- Michael D. add all work orders
    - 1NEW: Parent & Child
        - He assigns the assets
    - Permit is in there
- 2INPR: Inspectors goes into the field and inspects what work needs to be done
    - determines the asset that need to be replaced
    - SBO Inspectors in your group
    - Pictures in Maximo
    - Adds dimensions
    - 
- 2OKTS: Ok To Schedule
- 2HOLD: Sends it back, change order for another asset
- Contractors can get into the road
- Asset change order, get it approved by Austin Water

ROW PACT

- to see if they can do their work
- utility cuts and schedule the work

He doesn’t see the ticket until the work is completed

## More than one asset

- determines on how many children assets there are
- Asphalt - 2nd
- Concrete - 1st (wait a day or two then do follow-up to do Asphalt)
- Curb & Gutter (can do in one day)
- Combination work

Tells the UER team what kind of work needs to be done

- Sometimes it’s combined and it’s separated
- depends on the inspector
    - asphalt and concrete cut
    - needs pavers (specialty material)
    

Austin Water SR 

- How much work is given to UER?
    - 100%
- Basically paid for AW
- Seal Coat, Watershed

- How is work scheduled and issued out
    - Concrete (1st)
    - Asphalt (2nd)

## Permitting
- AW permits - 30 days
    - They don’t use Maximo
- They get tickets have 28 - 29 days
- The permits were about to expire, gave it to them without enough days
- Should have 26-28 days, not every ticket
- Some tickets, some permits will expire in a day or two

## Creates permits
Mike uses AMANDA AB+C Portal

- 30 minutes to create it
- line by line
- 15 page permit request
- when start/end dates
- How long does it take to get the permit?
    - right of way
    - wait for them to “Approve” them
- get and email the same day or next day
- secondary permit
- Arterial permit
    - high priority and high traffic
- Engineering plans
    - last one: 3 months to get permit plans
    - User a contractor for plans?
        - Sidewalks
        - Find a contractor to do it cheaper
        - time consuming

Rolling permits

- 2 weeks worth of work with permits
- Let permits expire and then gets on Joe H. schedule out

### PROCESS - Implementation
All the locations

150-160 locations for the next month

Map app? 

iPads - schedule optimized routing

- “Circuit” app
- Used to use the Map book, old school way
- Able to help other crews by mapping their work

### Daily Work Load amounts
- Asphalt: 8 to 10 work orders
- Concrete: 7-8, giving more than 8 work orders is a lot

Supervisor is looking at work order and giving estimates

- makes level of effort estimates based on work type

How many crews? 

- 1 asphalt: 10-12 people
- 1 big crew: 6 trucks, 6 crews (2 ppl crews)

### PROCESS - Field Work
- Use Weekly Schedule - spreadsheet
- Joe looks at them every day
- He changes the status “green”
    - 4COMP - in spreadsheet
- Email/Text Padilla - the numbers
    - He’ll review it, 4QACOMP
- Who is reporting on the work and what are they tracking?
    - Crews filling out tickets?
    - Use cell phones to take pictures to go into Maximo
    - Text crew leader when they’re done
    - Travel log in truck
    - Crew lead fills it in the office
        - First crew leader, set up, break out,
        - Second crew leader, how much they poured
    - Reconciles at the end of the day

### Maximo system performance and functionality
- iPads can be slow, depending on service and location
- Uses EzMaxMobile? 2n1 hardware issued out

- *How long is it taking to do a Maximo work order in the field?*

### PROCESS - Review & Reporting
Report back to Austin Water

- Send them bill
- Reimbursable process here
- Charges materials, hours, send to finance
- All staff UER, bills to a task order number - goes to AW
- Labor - timesheets
- Supervisors send to material tickets
    - Email to finance weekly
    - Send monthly to AW
    

They do not use contractors for this kind of work, they used to.

### User feedback and improvement

- ….

### Future roadmap and upgrades

- …

### Points of Contacts

- John P.
- Michael D.
- Joe H.

**Comment by ChristinaTremel** [comment]
Added notes to miro board and will review with the team next week!

**Comment by ChristinaTremel** [comment]
We met on 6/10 and processed all of the notes on the Miro board!
